### PR TITLE
EASYOPAC-472 - Add fix for carousel.

### PR DESCRIPTION
**Pull request by mihaelad**

### CODE REVIEW

File: modules/p2/ding_serendipity/css/ding_serendipity.css [hunk 1]
```diff
@@ -1,4 +1,6 @@
-.serendipity-admin-fieldset .fieldset-wrapper > div {
+.pane-serendipity-ting-object .ting-object {
+  
+  }.serendipity-admin-fieldset .fieldset-wrapper > div {
```
Comment (guddo) [inline modules/p2/ding_serendipity/css/ding_serendipity.css]: Missing new line?

File: themes/ddbasic/sass/module/_carousel.scss [hunk 2]
```diff
@@ -461,3 +453,27 @@ $carousel-tab-color:                       $white;
     font-size: 80%;
   }
 }
+
+.content-wrapper {
```
Comment (guddo) [inline themes/ddbasic/sass/module/_carousel.scss]: This is too "broad" a selector, please narrow it.

Comment (mihaelad) [inline themes/ddbasic/sass/module/_carousel.scss]: We try to override this style: https://github.com/easySuite/ding2/blob/development/themes/ddbasic/css/ddbasic.styles.css#L3468.

The only other solution is to add _!important_ rule. I don't think that it is ok.

Comment (guddo) [inline themes/ddbasic/sass/module/_carousel.scss]: Ok, I agree. It's a "global" selector. No problem :)
Remove the empty selector in the beginning of the file and all is  OK 👍

**Pull request merged**